include/Song.h

```diff
@@ -97,6 +97,7 @@ class EXPORT Song : public TrackContainer
 
 
 	void processNextBuffer();
+	void processAutomations(const TrackList& tracks, MidiTime timeStart, fpp_t frames, int tcoNum);
```

Comment:
If this is really private, you [should not](http://stackoverflow.com/a/7394311) be testing private methods. You should publish when necessary.